components/automate-deployment/pkg/services/internal/generated/gen.go

```diff
@@ -389,6 +389,16 @@ var ProductMetadataJSON = `
         ]
       }
     },
+    {
+      "name": "chef/automate-cs-ocid",
+      "metadata": {
+        "name": "chef/automate-cs-ocid",
+        "data_service": false,
+        "binlinks": null,
+        "uses_platform_scaffolding": true,
+        "bootstrap": null
```

Comment:
need to add the changes here for secret management

Reply:
@punitmundra  Can you please provide further context for this?

What is  secret management in context of Automate and how should OCID use it?